pkg/artifacts/artifacts_storage.go

```diff
@@ -173,7 +178,14 @@ func createPVC(pr *v1alpha1.PipelineRun, c kubernetes.Interface) (*corev1.Persis
 			if err != nil {
 				return nil, xerrors.Errorf("failed to create Persistent Volume spec for %q due to error: %w", pr.Name, err)
 			}
-			pvcSpec := GetPVCSpec(pr, pvcSize)
+			var pvcStorageClassName *string
+			if pvcStorageClassNameStr == "" {
+				pvcStorageClassName = nil
```

Comment:
Since this goes to `nil` if it's not defined, it will work fine on existing (or new) clusters that do not include the storageClass in the config map.